src/main/resources/wa-task-types-publiclaw-care_supervision_epo.dmn

```diff
@@ -229,6 +284,17 @@
           <text>"Chase Outstanding Order"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0972qcs">
+        <inputEntry id="UnaryTests_17em3ml">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_006764i">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_11rgq3v">
+          <text></text>
+        </outputEntry>
+      </rule>
```

Comment:
empty row at the bottom of the task type DMN - should remove I think (sometimes they cause issues, not sure if this one would specifically but best to remove anyway)